capa/rules/cache.py

```diff
@@ -173,8 +269,7 @@ def generate_rule_cache(rules_dir: Path, cache_dir: Path) -> bool:
         logger.error("%s", str(e))
         return False
 
-    content = capa.rules.cache.get_ruleset_content(rules)
-    id = capa.rules.cache.compute_cache_identifier(content)
```

Comment:
sure to remove this?

Reply:
Yeah, becasue `get_ruleset_content` is now called in `compute_ruleset_cache_identifier`. We can always move it out and leave it here, as part of `generate_rule_cache`.

Reply:
ah I see